### PR TITLE
Add info box to alert users to change their permissions

### LINKED ISSUE #322: Private group assignment advisory
On the private group assignment page we need to add an info box telling the user to set their "Default repository permission" to "None" so that students can't see each other's assignments.

The info box should link to the exact settings page for that organization.

### PR DESCRIPTION
**Pull request by tarebyte**
Closes #322 

screen shot 2015-11-02 at 11 20 52 am

/cc @johndbritton

### CODE REVIEW

**Comment by johndbritton** [comment]
Is there a way for us to show this message conditionally depending on the default repository permission setting? If so, we should only show this in the case that it's not already set to "None."

File: app/views/group_assignments/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <%= render 'organizations/organization_banner' %>
 
+<div class="flash flash-warn">
+  <%= octicon('broadcast') %>
+  Please go to your organization <%= link_to 'settings', "https://github.com/organizations/#{organization.login}/settings/member_privileges", target: "_blank" %> and update your <b>Default repository permission</b> to none.
```
Comment (johndbritton) [inline app/views/group_assignments/show.html.erb]: We should tell the user why they need to do this.

**Comment by tarebyte** [comment]
This is the new box and the working functionality.
permission box

**Comment by johndbritton** [comment]
:+1: :shipit:

**Pull request merged**